File: PHP/DragaoTest.php
<?php
/**
 * @version PHP
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 * $ ./vendor/bin/phpunit NomeDaClasseTest.php
 */
namespace PHP;

use PHPUnit\Framework\TestCase;
require ("Dragao.php");

class DragaoTest extends TestCase {

    public function testDragao() {
        $obj = new Dragao();
        //Test 1
        $want = "Mais de 8000!";
        $got = $obj->checkEnergyLevel(8001);
        $this->assertEquals($want, $got);
        //Test 2
        $want = "Inseto!";
        $got = $obj->checkEnergyLevel(100);
        $this->assertEquals($want, $got);
        //Test 3
        $want = "Inseto!";
        $got = $obj->checkEnergyLevel(200);
        $this->assertEquals($want, $got);
    }

}

File: JAVA/DragaoTest.java
/** 
 * @version JAVA
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 * $ mvn clean test -Dtest=your.package.TestClassName
 */ 
package JAVA;

import org.junit.Assert;
import org.junit.Test;

public class DragaoTest {

	@Test
	public void testDragao() throws Exception {
		//Test 1
		String want = "Mais de 8000!";
		String got = Dragao.checkEnergyLevel(8001);
        Assert.assertEquals(want, got);
		//Test 2
		want = "Inseto!";
		got = Dragao.checkEnergyLevel(100);
        Assert.assertEquals(want, got);
		//Test 3
		want = "Inseto!";
		got = Dragao.checkEnergyLevel(200);
        Assert.assertEquals(want, got);
	}

}
